setup/pi/create-backingfiles.sh

```diff
@@ -78,10 +78,14 @@ function add_drive () {
   fallocate -l "$size"K "$filename"
   echo "type=c" | sfdisk "$filename" > /dev/null
   local partition_offset=$(first_partition_offset "$filename")
-  losetup -o $partition_offset loop0 "$filename"
+  local next_dev_loop="$(losetup -f | cut -f3 -d /)"
```

Comment:
the 'cut' isn't actually needed, the losetup command will take a "/dev/loopXXX" style argument as well.